src/concrete/ml/onnx/onnx_utils.py

```diff
@@ -461,9 +464,13 @@ def execute_onnx_with_numpy(
             for initializer in graph.initializer
         },
     )
+
     for node in graph.node:
         curr_inputs = (node_results[input_name] for input_name in node.input)
         attributes = {attribute.name: get_attribute(attribute) for attribute in node.attribute}
+
+        if node.op_type in ["Less", "LessOrEqual", "Greater", "GreaterOrEqual"]:
```

Comment:
I think you should add a global constant list with these and put it next to `ONNX_OPS_TO_NUMPY_IMPL_BOOL`